app/src/interfaces/select-color/select-color.vue

```diff
@@ -63,8 +105,41 @@
 					maxlength="3"
 					@update:model-value="setValue('hsl', i, $event)"
 				/>
+				<v-input
+					v-if="opacity"
+					type="number"
+					:model-value="alpha"
+					class="color-data-input"
+					pattern="\d*"
+					:min="0"
+					:max="100"
+					:step="1"
+					maxlength="3"
+					@update:model-value="setValue('alpha', 0, $event)"
+				/>
 			</template>
 		</div>
+		<div v-if="opacity" class="color-data-alphas">
+			<div class="color-data-alpha">
+				<v-slider
+					:model-value="alpha"
+					:min="0"
+					:max="100"
+					:step="1"
+					:style="{
+						'--v-slider-color': 'none',
+						'--background-page': 'none',
+						'--v-slider-fill-color': 'none',
+						'--v-slider-thumb-color': 'var(--foreground-normal)',
+						'--input-bg-image':
+							'linear-gradient(to right, transparent,' +
+							(hex && hex.length === 9 ? hex.slice(0, -2) : hex ? hex : 'transparent') +
+							'), url(\'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAMUlEQVQ4T2NkYGAQYcAP3uCTZhw1gGGYhAGBZIA/nYDCgBDAm9BGDWAAJyRCgLaBCAAgXwixzAS0pgAAAABJRU5ErkJggg==\')',
```

Comment:
Great solution here! One little nitpick, lets rename `--input-bg-image` to `--v-slider-track-background-image`.